app/models/bot.rb

```diff
@@ -1,8 +1,9 @@
 require 'indico'
+require 'csv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.